### PR TITLE
Feature/EFRS-925 - fix lint errors and format code

### PR DESCRIPTION
**Pull request by isobolev**
- added npm script to format all;
- formatted code;
- fixed lint errors; 
- organised imports; 
- added copyright header, where missing;
- removed unused files

### CODE REVIEW

File: ui/src/app/features/model-table/model-table.component.spec.ts [hunk 1]
```diff
@@ -13,17 +13,16 @@
...
-
-import { ModelTableComponent } from './model-table.component';
-import { TranslatePipe, TranslateService } from '@ngx-translate/core';
 import { MatTooltipModule } from '@angular/material/tooltip';
+import { TranslatePipe, TranslateService } from '@ngx-translate/core';
+
 import { TruncateModule } from '../../ui/truncate-pipe/truncate.module';
+import { ModelTableComponent } from './model-table.component';
 
 xdescribe('ModelTableComponent', () => {
```
Comment (azanorth) [inline ui/src/app/features/model-table/model-table.component.spec.ts:27]: It looks like that test was excluded

File: ui/src/app/pages/demo/demo.guard.spec.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+import { of } from 'rxjs';
 
 import { DemoGuard } from './demo.guard';
-import { provideMockStore } from '@ngrx/store/testing';
 import { DemoService } from './demo.service';
-import { of } from 'rxjs';
-import { Store } from '@ngrx/store';
-import { Router } from '@angular/router';
 
 xdescribe('DemoGuard', () => {
```
Comment (azanorth) [inline ui/src/app/pages/demo/demo.guard.spec.ts:10]: It looks like that test was excluded

**Review by azanorth — COMMENTED** [review]
Looks good

File: ui/src/app/features/model-table/model-table.component.spec.ts (hunk 1 above)
Comment (isobolev) [reply ui/src/app/features/model-table/model-table.component.spec.ts:27]: good spot, not part of the changes in this PR though. let's fix those tests in another PR

File: ui/src/app/pages/demo/demo.guard.spec.ts (hunk 2 above)
Comment (isobolev) [reply ui/src/app/pages/demo/demo.guard.spec.ts:10]: same as above

**Review by azanorth — APPROVED** [review]

**Pull request merged**